odata_client/src/test/java/com/sdl/odata/client/caller/BasicEndpointCallerTest.java

```diff
@@ -56,6 +57,7 @@
 @ExtendWith(SpringExtension.class)
 @SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT, classes = TestConfiguration.class)
 @DirtiesContext
+@Disabled
```

Comment:
Do we need a comment/TODO to reenable it?

Reply:
it's captured on confluence in a TODO list to fix later